src/huggingface_hub/hf_api.py

```diff
@@ -9564,6 +9613,7 @@ def _parse_revision_from_pr_url(pr_url: str) -> str:
 list_repo_commits = api.list_repo_commits
 list_repo_tree = api.list_repo_tree
 get_paths_info = api.get_paths_info
+auth_check = api.auth_check
```

Comment:
(nit) could you move this line above between `whoami = api.whoami` and `get_token_permission = api.get_token_permission`? Thanks!